CMakeLists.txt

```diff
@@ -184,10 +185,20 @@ mark_as_advanced(BUILD_TESTS)
 option(BUILD_ASE_TESTS "Enable ASE test compilation." OFF)
 mark_as_advanced(BUILD_ASE_TESTS)
 
+## TODO: Remove this once this replaces old school tests
+option(NEW_UNITS "Use new testing framework" OFF)
+mark_as_advanced(NEW_UNITS)
+option(ENABLE_MOCK "Enable mock driver for testing." ON)
+mark_as_advanced(ENABLE_MOCK)
+
 if(BUILD_TESTS OR BUILD_ASE_TESTS)
   enable_testing()
   include(CTest)
-  add_subdirectory(tests)
+  if (NEW_UNITS)
```

Comment:
Maybe we can remove NEW_UNITS option and replace this line with BUILD_TESTS

Reply:
I think so.